src/Bling.js

```diff
@@ -229,8 +229,7 @@ class Bling extends Component {
         "collapseEmptyDiv",
         "companionAdService",
         "forceSafeFrame",
-        "safeFrameConfig",
-        "limitedAds"
+        "safeFrameConfig"
```

Comment:
Are those changes from the upstream repo?

Reply:
Nope, this was one of the initial issues. At the initial render shouldRefresh should be false but could be triggered due to this. These fields are ours additions.